src/ImageSharp/Primitives/BokehBlurParameters.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.ImageSharp.Primitives
+{
+    /// <summary>
+    /// A <see langword="struct"/> that contains parameters to apply a bokeh blur filter
+    /// </summary>
+    internal readonly struct BokehBlurParameters
```

Comment:
Implement `IEquatable<T>` etc  for best practice

Reply:
I don't think this an any other BokehBlur specific primitives should live under this namespace. `ComplexVector` and `Complex64` are an exception.

Reply:
Done in https://github.com/SixLabors/ImageSharp/pull/842/commits/9edf46b511c363839c3241ff38bb401c505ae8fc and https://github.com/SixLabors/ImageSharp/pull/842/commits/db581635595e6db4dff75fbe726bfc67f51b8188.
I've moved the two types into a `Parameters` folder in the `Convolution` processors subfolders. We can chage this later on if you want.